include/boost/gil/extension/io/targa/detail/supported_types.hpp

```diff
@@ -107,4 +107,4 @@ struct is_write_supported< Pixel
 } // namespace boost
 
 
-#endif
+#endif // BOOST_GIL_EXTENSION_IO_TARGA_DETAIL_SUPPORTED_TYPES_HPP
```

Comment:
Is this really necessary ? One more thing to track should we ever rename or move the file again...
(I was just thinking we might just replace include guards by `#pragma once` statements, to avoid the debate entirely :-)

Reply:
I think I got warned by an editor (VSCode?), some kind of linting annoyance. Unnecessary indeed. I'll remove it.


I'm all for `#pragma once`!

Question: is it allowed/forbidden by Boost?

However, we should be aware and agree on caveats:
* [It was considered for standardization, but rejected because it cannot be implemented reliably](https://stackoverflow.com/a/23699893/151641)
* It may be troublesome in some [rare convoluted cases](https://en.wikipedia.org/wiki/Pragma_once#Caveats).
* We agree we don't have to support any compilers not included [in this table](https://stackoverflow.com/q/23696115/151641) 

/cc @chhenning